root/robots.txt.production

```diff
@@ -19,15 +21,36 @@ Disallow: /register
 Disallow: /report/
 Disallow: /reports
 Disallow: /search
+Disallow: /set-beta-preference
+Disallow: /statistics/timeline
```

Comment:
How come? We don't really link it anywhere except the menu and it seems useful to show up in searches for "musicbrainz timeline" etc.